capa/features/extractors/elffile.py

```diff
@@ -36,10 +53,13 @@ def extract_file_import_names(elf, **kwargs):
         logger.debug("Symbol table '%s' contains %s entries:", section.name, section.num_symbols())
 
         for _, symbol in enumerate(section.iter_symbols()):
-            if symbol.name and symbol.entry.st_info.type == "STT_FUNC":
-                # TODO(williballenthin): extract symbol address
-                # https://github.com/mandiant/capa/issues/1608
-                yield Import(symbol.name), FileOffsetAddress(0x0)
+            # The following conditions are based on the following article
+            # http://www.m4b.io/elf/export/binary/analysis/2015/05/25/what-is-an-elf-export.html
+            if symbol.name and symbol.entry.st_info.type in ["STT_FUNC", "STT_OBJECT", "STT_IFUNC"]:
+                if symbol.entry.st_value == 0 and symbol.entry.st_shndx == "SHN_UNDEF" and symbol.entry.st_name != 0:
+                    # TODO(williballenthin): extract symbol address
+                    # https://github.com/mandiant/capa/issues/1608
+                    yield Import(symbol.name), FileOffsetAddress(0x0)
```

Comment:
```suggestion
            if not symbol.name:
                continue
            if symbol.entry.st_info.type not in ["STT_FUNC", "STT_OBJECT", "STT_IFUNC"]:
                continue
            if symbol.entry.st_value != 0:
                continue
            if symbol.entry.st_shndx != "SHN_UNDEF":
                continue
            if symbol.entry.st_name == 0:
                continue

            # TODO(williballenthin): extract symbol address
            # https://github.com/mandiant/capa/issues/1608
            yield Import(symbol.name), FileOffsetAddress(0x0)
```

Reply:
would you also be able to address the TODO for #1608 here, too?

Reply:
Will try it ? Currently looking into how to get file offset address of import symbol name  in elffile.
I think the address we want to get should be the import symbol name location in elf file.

Reply:
do you want to address this TODO in the current PR? or should I merge it and you can open a new PR with the fix?

Reply:
I will be doing it in this one. It will be done in a few minutes.